### PR TITLE
LRAC-8324 | Create stubs test for Property Analytics Filter

### PR DESCRIPTION
**Pull request by LucasAcacio**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		ACUtils.tearDownDXP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Automation ID: Legacy | Automation ID: LRAC- 8319 | Test Summary: "
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:32]: Hi @LucasAcacio , there is an extra space here.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	@ignore = "true"
+	@priority = "5"
+	test CustomRangeTimePeriodFilterSites {
+
+		// TODO LRAC- 8319
+
+	}
+
+	@description = "Automation ID: Legacy | Automation ID: LRAC- 8321 | Test Summary: "
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:41]: Hi @LucasAcacio , the description is not following the pattern.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	@ignore = "true"
+	@priority = "4"
+	test FirstAccessAndNoProperty {
+
+		// TODO LRAC- 8321
+
+	}
+
+	@description = "Automation ID: Legacy | Automation ID: LRAC- 8317 | Test Summary: "
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:50]: hi @LucasAcacio , the Test Summary is empty.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+		// TODO LRAC- 8319
+
+	}
+
+	@description = "Automation ID: Legacy | Automation ID: LRAC- 8321 | Test Summary: "
+	@ignore = "true"
+	@priority = "4"
+	test FirstAccessAndNoProperty {
+
+		// TODO LRAC- 8321
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:46]: hi @LucasAcacio , there is extra space on the ticket ID.

**Comment by alexcandidos** [comment]
hi @LucasAcacio , could you please take a look at my comments? Thanks!

**Comment by LucasAcacio** [comment]
Required changes done

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**